app/models/theme_field.rb

```diff
@@ -253,6 +254,24 @@ def target_name
     MessageBus.publish "/header-change/#{theme.id}", self.value if theme && self.name == "header"
     MessageBus.publish "/footer-change/#{theme.id}", self.value if theme && self.name == "footer"
   end
+
+  private
+
+  JAVASCRIPT_TYPES = %w(
+    text/javascript
+    application/javascript
+    application/ecmascript
+  )
```

Comment:
This is not an exhaustive list -- only the most common ones and current standard, according to https://developer.mozilla.org/en-US/docs/Web/HTML/Element/script#attr-type